locale/en/blog/community/v5-to-v7.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Farewell to Node.js v5, Preparing for v7
+date: 2016-08-30T06:57:53.463Z
+status: publish
+category: Annoucements
+slug: v5-to-v7
+layout: blog-post.hbs
+author: Rod Vagg
+---
+
+You may have missed it but at the end of June, the Node.js project said a final farewell to version 5. There will be no more patches, critical or otherwise, for this branch. To those who have been using Node.js for some time this may seem anomalous, shouldn't major versions stick around for _years_?
+
+## We have a plan!
+
+![LTS Schedule Summary](/static/images/blog/201609_lts_schedule_summary.gif)
+
+Last year, the core team devised an LTS and release plan to balance the various wants and needs expressed by Node.js users. Chief among those were:
+
+1. Stability
+2. Progress
+
+The io.js diversion was useful for many reasons, including the opportunity we had to lean into this "progress" thing. We learned that there is a necessary trade-off between "stability" and the rapid iteration of the platform. Some of it was manageable but much was unavoidable. Breaking the entire C++ add-on ecosystem each time we upgraded V8 turned out to be quite painful for the Node.js package ecosystem. This is due to the heavy reliance on compiled native components in Node.js userland and the V8 project not having to be too concerned with the breakage and pain caused by the constant [API](wikipedia API) and [ABI](wikipedia ABI) churn since its primary consumer is Chromium.
```

Comment:
Sorry, I didn't intend this as mud slinging. It's an objective observation based on a lot of history and accurately reflects the state we were in during io.js, even if things have improved lately. I'll adjust and ping you both when I have an update.